### ISSUE TITLE
Does not build on MIPS and ARMv5

### ISSUE THREAD

**Issue by messense** [metrics-rs/quanta#54]
MIPS issue: https://github.com/metrics-rs/quanta/issues/22#issuecomment-914420194 , MIPS seems work before https://github.com/metrics-rs/quanta/pull/52

ARMv5(armv5te-unknown-linux-musleabi target) has the similar issue

```
error[E0599]: no method named `fetch_add` found for struct `Arc<AtomicCell<u64>>` in the current scope
  --> /root/.cargo/registry/src/github.com-1ecc6299db9ec823/quanta-0.9.2/src/mock.rs:48:21
   |
48 |         self.offset.fetch_add(amount.into_nanos());
   |                     ^^^^^^^^^ method not found in `Arc<AtomicCell<u64>>`

error[E0599]: no method named `fetch_sub` found for struct `Arc<AtomicCell<u64>>` in the current scope
  --> /root/.cargo/registry/src/github.com-1ecc6299db9ec823/quanta-0.9.2/src/mock.rs:53:21
   |
53 |         self.offset.fetch_sub(amount.into_nanos());
   |                     ^^^^^^^^^ method not found in `Arc<AtomicCell<u64>>`

error: aborting due to 2 previous errors
```

**Comment by paolobarbolini** [comment]
Looks like crossbeam doesn't expose those functions when the target doesn't support 64 bit atomics https://docs.rs/crossbeam-utils/0.8.5/src/crossbeam_utils/atomic/atomic_cell.rs.html#540
Either that can be changed in crossbeam or those operations will have to be implemented with `fetch_update`

**Comment by messense** [comment]
Alternatively can we put the time mocking feature under a on-by-default `mock` feature gate? I think it's mostly used in tests?

**Comment by messense** [comment]
@tobz Opened https://github.com/metrics-rs/quanta/pull/55

**Comment by tobz** [comment]
I'll cut a new release with #55 in it later today.

**Comment by taiki-e** [comment]
fwiw, this was a bug/regression of crossbeam-utils, and fixed in crossbeam-utils 0.8.6 (https://github.com/crossbeam-rs/crossbeam/pull/767).
